packages/toolkit/src/view/pipeline-builder/nodes/CustomNode.tsx

```diff
@@ -136,48 +136,52 @@ export const Root = React.forwardRef<
             }
           
           */}
-          {["texts", "images", "structured_data", "metadata"].map((e) => (
-            <div
-              key={`${nodeId}-${e}`}
-              className="flex px-2 flex-row py-4 odd:bg-semantic-bg-base-bg even:bg-semantic-secondary-bg last:rounded-br-sm last:rounded-bl-sm"
-            >
-              <Handle
-                className={cn(
-                  "!relative !my-auto !top-none !left-none !transform-none !w-3 !h-3 !border !box-content !border-semantic-bg-line",
-                  edges.some((edge) => edge.targetHandle === `${nodeId}.${e}`)
-                    ? "!bg-[#595959]"
-                    : "!bg-[#94A0B8]",
-                  {
-                    invisible:
-                      connectorDefinitionName ===
-                      "connector-definitions/trigger",
-                  }
-                )}
-                type="target"
-                position={Position.Left}
-                id={`${nodeId}.${e}`}
-              />
+          {["texts", "images", "structured_data", "metadata", "audios"].map(
```

Comment:
I think the metadata should be at the bottom of the node so the array will looks like texts, images, audios, structured_data, metadata

Reply:
updated the suggested changes.